README.md

```diff
@@ -41,11 +41,34 @@ Attribute service fetches all attributes relevant to the scope of what is being
 The Attribute service uses gradlew to compile/install/distribute. Gradle wrapper is already part of the source code. To build Attribute Service, run:
 
 ```
-./gradlew clean build dockerBuildImages
+./gradlew dockerBuildImages
 ```
 
+## Testing
+
+### Running unit tests
+Run `./gradlew test` to execute unit tests. 
+
+
+### Testing image
+
+To test your image using the docker-compose setup follow the steps:
+
+- Commit you changes to a branch say attribute-service-test
+- Go to [hypertrace-service](https://github.com/hypertrace/hypertrace-service) and checkout the above branch in the submodule.
```

Comment:
Can you give the command too. Something like the below? Not sure if that is accurate.
`cd attribute-service && git checkout attribute-service-test && cd ..`

Reply:
Okay!